server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfile.java

```diff
@@ -28,9 +31,15 @@ public MemberProfile(String name, String role, UUID pdlId, String location,
                         this.bioText=bioText;
                         }
 
-    @Id
-    @Column(name="uuid")
-    @GeneratedValue
+    public MemberProfile() {
+	}
+
+	@Id
+    @Type(type = "uuid-char")
+    @Column(name="uuid", updatable = false, nullable = false, unique = true)
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    @ColumnDefault("random_uuid()")
```

Comment:
Interesting...does this impl work with both H2 and PostgreSQL?

Reply:
I am unsure since I don't have PostgreSQL setup on my local yet. Here's a blog that explains config with PostgreSQL - https://vladmihalcea.com/uuid-identifier-jpa-hibernate/